common/src/parse.cpp

```diff
@@ -181,14 +181,16 @@ parse_argument (int argc, const char * const * argv, const char * str, T &val) n
 int
 pcl::console::parse_argument (int argc, const char * const * argv, const char * str, double &val)
 {
-  return parse_generic(strtod, argc, argv, str, val);
+  const auto strtod_l = [](const char *str, char **str_end){ return strtod(str, str_end); };
```

Comment:
Comment for the reason

Reply:
Thanks for your advice! I'll fix it right now! :grinning: